tsktsk/task.py

```diff
@@ -1,23 +1,37 @@
 import dataclasses
 import textwrap
 from datetime import datetime
+from enum import Enum
 from typing import Optional
 
-CATEGORY_DEFAULT = "NEW"
-VALUE_DEFAULT = "medium"
-EFFORT_DEFAULT = "medium"
 
-CATEGORY = {
-    "NEW": "📦 NEW",
-    "IMP": "👌 IMP",
-    "FIX": "🐛 FIX",
-    "DOC": "📖 DOC",
-    "TST": "✅ TST",
-}
+class Category(Enum):
+    NEW = "📦 NEW"
+    IMP = "👌 IMP"
+    FIX = "🐛 FIX"
+    DOC = "📖 DOC"
+    TST = "✅ TST"
 
-VALUE = {"high": "V⬆", "medium": "", "low": "V⬇"}
-EFFORT = {"high": "E⬆", "medium": "", "low": "E⬇"}
-POINTS = {"high": 8, "medium": 5, "low": 3}
+    DEFAULT = NEW
+
+
+class Value(Enum):
+    HIGH = "V⬆"
+    MEDIUM = ""
+    LOW = "V⬇"
+
+    DEFAULT = MEDIUM
+
+
+class Effort(Enum):
+    HIGH = "E⬆"
+    MEDIUM = ""
+    LOW = "E⬇"
+
+    DEFAULT = MEDIUM
+
+
+POINTS = {"HIGH": 8, "MEDIUM": 5, "LOW": 3}
```

Comment:
(Random comment, not request for change)
Because of this type of dict, I had considered making an enum `Estimate`, for the high/medium/low, and category/value would be of type Estimate. I think it would make some things easier and something harder tho.

Reply:
Yeah, I originally had an Estimate enum. but eventually, decided to go with separate enums. It led to a lot of duplications that I didn't like and found working with separate enums easier.